### ISSUE TITLE
Include license in release files

### ISSUE THREAD

**Issue by arvoelke** [arvoelke/nengolib#116]
Currently has the readme but not the license. Need to investigate.

For reference, here are the current contents of the `v0.4.0-beta` tarball:

```
nengolib-0.4.0b0/
nengolib-0.4.0b0/nengolib/
nengolib-0.4.0b0/nengolib/compat.py
nengolib-0.4.0b0/nengolib/conftest.py
nengolib-0.4.0b0/nengolib/connection.py
nengolib-0.4.0b0/nengolib/monkeypatch.py
nengolib-0.4.0b0/nengolib/network.py
nengolib-0.4.0b0/nengolib/networks/
nengolib-0.4.0b0/nengolib/networks/echo_state.py
nengolib-0.4.0b0/nengolib/networks/linear_network.py
nengolib-0.4.0b0/nengolib/networks/reservoir.py
nengolib-0.4.0b0/nengolib/networks/rolling_window.py
nengolib-0.4.0b0/nengolib/networks/tests/
nengolib-0.4.0b0/nengolib/networks/tests/test_echo_state.py
nengolib-0.4.0b0/nengolib/networks/tests/test_linear_network.py
nengolib-0.4.0b0/nengolib/networks/tests/test_reservoir.py
nengolib-0.4.0b0/nengolib/networks/tests/test_rolling_window.py
nengolib-0.4.0b0/nengolib/networks/tests/__init__.py
nengolib-0.4.0b0/nengolib/networks/__init__.py
nengolib-0.4.0b0/nengolib/neurons.py
nengolib-0.4.0b0/nengolib/processes.py
nengolib-0.4.0b0/nengolib/signal/
nengolib-0.4.0b0/nengolib/signal/discrete.py
nengolib-0.4.0b0/nengolib/signal/dists.py
nengolib-0.4.0b0/nengolib/signal/learning.py
nengolib-0.4.0b0/nengolib/signal/lyapunov.py
nengolib-0.4.0b0/nengolib/signal/realizers.py
nengolib-0.4.0b0/nengolib/signal/reduction.py
nengolib-0.4.0b0/nengolib/signal/system.py
nengolib-0.4.0b0/nengolib/signal/tests/
nengolib-0.4.0b0/nengolib/signal/tests/test_discrete.py
nengolib-0.4.0b0/nengolib/signal/tests/test_dists.py
nengolib-0.4.0b0/nengolib/signal/tests/test_learning.py
nengolib-0.4.0b0/nengolib/signal/tests/test_lyapunov.py
nengolib-0.4.0b0/nengolib/signal/tests/test_realizers.py
nengolib-0.4.0b0/nengolib/signal/tests/test_reduction.py
nengolib-0.4.0b0/nengolib/signal/tests/test_system.py
nengolib-0.4.0b0/nengolib/signal/tests/test_utils.py
nengolib-0.4.0b0/nengolib/signal/tests/__init__.py
nengolib-0.4.0b0/nengolib/signal/utils.py
nengolib-0.4.0b0/nengolib/signal/__init__.py
nengolib-0.4.0b0/nengolib/solvers.py
nengolib-0.4.0b0/nengolib/stats/
nengolib-0.4.0b0/nengolib/stats/leech.py
nengolib-0.4.0b0/nengolib/stats/ntmdists.py
nengolib-0.4.0b0/nengolib/stats/ortho.py
nengolib-0.4.0b0/nengolib/stats/tests/
nengolib-0.4.0b0/nengolib/stats/tests/test_leech.py
nengolib-0.4.0b0/nengolib/stats/tests/test_ntmdists.py
nengolib-0.4.0b0/nengolib/stats/tests/test_ortho.py
nengolib-0.4.0b0/nengolib/stats/tests/__init__.py
nengolib-0.4.0b0/nengolib/stats/_sobol_seq.py
nengolib-0.4.0b0/nengolib/stats/__init__.py
nengolib-0.4.0b0/nengolib/synapses/
nengolib-0.4.0b0/nengolib/synapses/analog.py
nengolib-0.4.0b0/nengolib/synapses/digital.py
nengolib-0.4.0b0/nengolib/synapses/hetero_synapse.py
nengolib-0.4.0b0/nengolib/synapses/mapping.py
nengolib-0.4.0b0/nengolib/synapses/tests/
nengolib-0.4.0b0/nengolib/synapses/tests/test_analog.py
nengolib-0.4.0b0/nengolib/synapses/tests/test_digital.py
nengolib-0.4.0b0/nengolib/synapses/tests/test_hetero_synapse.py
nengolib-0.4.0b0/nengolib/synapses/tests/test_mapping.py
nengolib-0.4.0b0/nengolib/synapses/tests/__init__.py
nengolib-0.4.0b0/nengolib/synapses/__init__.py
nengolib-0.4.0b0/nengolib/tests/
nengolib-0.4.0b0/nengolib/tests/test_connection.py
nengolib-0.4.0b0/nengolib/tests/test_monkeypatch.py
nengolib-0.4.0b0/nengolib/tests/test_network.py
nengolib-0.4.0b0/nengolib/tests/test_neurons.py
nengolib-0.4.0b0/nengolib/tests/test_notebooks.py
nengolib-0.4.0b0/nengolib/tests/test_processes.py
nengolib-0.4.0b0/nengolib/tests/test_solvers.py
nengolib-0.4.0b0/nengolib/tests/__init__.py
nengolib-0.4.0b0/nengolib/version.py
nengolib-0.4.0b0/nengolib/__init__.py
nengolib-0.4.0b0/nengolib.egg-info/
nengolib-0.4.0b0/nengolib.egg-info/dependency_links.txt
nengolib-0.4.0b0/nengolib.egg-info/PKG-INFO
nengolib-0.4.0b0/nengolib.egg-info/requires.txt
nengolib-0.4.0b0/nengolib.egg-info/SOURCES.txt
nengolib-0.4.0b0/nengolib.egg-info/top_level.txt
nengolib-0.4.0b0/PKG-INFO
nengolib-0.4.0b0/README.rst
nengolib-0.4.0b0/setup.cfg
nengolib-0.4.0b0/setup.py
```

**Comment by arvoelke** [comment]
And include CHANGELOG in release files (#120).

**Comment by arvoelke** [comment]
Resolved by #120.
